staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch_test.go

```diff
@@ -2366,3 +2368,48 @@ func TestNumberConversion(t *testing.T) {
 		}
 	}
 }
+
+var replaceRawExtensionPatchTestCases = []StrategicMergePatchRawTestCase{
+	{
+		Description: "replace RawExtension field",
+		StrategicMergePatchRawTestCaseData: StrategicMergePatchRawTestCaseData{
+			Original: []byte(`
+replacingItem:
+  Some: Generic
+  Yaml: Inside
+  The: RawExtension
+  Field: Period
+other: current-other
```

Comment:
include enough other fields to be able to tell that computation of other aspects of the object are still working (original field unchanged, original field modified, etc)